### ISSUE TITLE
Avoid creating instanced scene copies that litter the filesystem

### ISSUE THREAD

**Issue by fire** [godotengine/godot-proposals#7173]
### Describe the project you are working on

VR Virtual worlds project with possibility of being open world and Godot Engine.

### Describe the problem or limitation you are having in your project

Imported scenes which have been modified by the developer will create thousands of instanced scene copies that litter the filesystem.

See also proposal https://github.com/godotengine/godot-proposals/issues/6932

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

1. Make sure the properties are expose to the importer See https://github.com/godotengine/godot/pull/78803
2. Find a way to merge the settings back into the `.import` file to eliminate the difference.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

* To be determined.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

It is not a few lines of script because a new script needs to be written for each type of import.

### Is there a reason why this should be core and not an add-on in the asset library?

We try to make developer's lives easier.

**Comment by EMBYRDEV** [comment]
Thank you!

**Comment by fire** [comment]
@EMBYRDEV I forgot something, did you know that if you rename an .tscn as `.escn` it'll reimport?

**Comment by lyuma** [comment]
I put a link in #78803 since I was told this was the proposal for that feature PR. The title is somewhat misleading since it is using plurals when in fact this proposal can only solve the case of a single modified scene copy. To be clear, after importing a scene, if customization of the imported scene is needed, only a single extra file per imported scene is needed, and it should be inhertied rather than instantiated. Does that make sense.

That is to say, when you import a .glb file, if you want to change mesh layers, the current workflow will create foo.glb, foo.glb.import and a foo.tscn which inherits the imported scene. This proposal would be to eliminate the .tscn by allowing properties changed in the .tscn to be saved as part of the imported scene.

Now, I'll try and generalize the points I made in a comment here: https://github.com/godotengine/godot/pull/78803#issuecomment-1670049937

As I noted in the above linked comment, this proposal is already implementable as an addon. This was the intended design behind the new EditorScenePostImportPlugin system, which permits exposing project-wide advanced importer settings, and to do post-processing on the scene of a similar nature to what can be done by inhertiance.

In short, I'm missing elements of the design. For "Expose the properties to the importer", does this mean that *all* node properties which could conceivably change in an inherited scene should be exposed?

**Comment by fire** [comment]
Yes, all modified scene properties as per our saving process

**Comment by akien-mga** [comment]
Is this fully addressed by https://github.com/godotengine/godot/pull/78803?

**Comment by lyuma** [comment]
In my opinion, this proposal has not been addressed.

#78803 added some helpful options commonly used by some users, but I think there are many more workflows which require creating inherited scenes that were not addressed.

That said, it would be helpful to hear from users about what their needs for inherited scenes are.

**Comment by EMBYRDEV** [comment]
> Is this fully addressed by [godotengine/godot#78803](https://github.com/godotengine/godot/pull/78803)?

I don't believe so either, this is a wider reaching problem that https://github.com/godotengine/godot/pull/78803 only partially addresses. 

There are a few more properties that are likely useful to expose but as @lyuma stated I'd like to hear some more suggestions.

I'm not alergic to the idea of creating inherited scenes to do very specific setup but I dont think it should be something we have to do by default.
